docs/api-docs/storefront/widgets/widget-builder.md

```diff
@@ -0,0 +1,187 @@
+? What is the Access Token? xxxxxxxxxxxxx
+? What is the API Path? https://api.bigcommerce.com/stores/xxxxx/v3/
+[2021-09-08T15:12:40.271Z] Successfully created your configuration, you're all set!
+```
+## Reset or create parallel configurations
+
+If you need to reset the configuration, you can run the `widget-builder init` command in the same directory to overwrite the existing assignments.  Running `widget-builder init` in sibling directory will create a separate development environment.  
+
+Initializing Widget Builder at multiple levels of a single file hierarchy can have unpredictable results and is not recommended.
+ 
+ 
+<div class="HubBlock--callout">
+<div class="CalloutBlock--warn">
+<div class="HubBlock-content">
+
+<!-- theme: warn -->
+
+> To use Widget Builder as a development tool, you must run `widget-builder` commmands from the same directory where you ran `widget-builder init`.
+
+</div>
+</div>
+</div>
+  
+## Generate a scaffold with `create`
+
+To jump start widget development, Widget Builder's `create` command creates a child directory, populates it with the blank template files you need to get started, and spins up a development server.  To use the generator, run the following command:
+
+```shell
+widget-builder create [[path-to/]some-widget]
+```
+
+**Sample output:**
+
+```shell
+[2021-09-08T17:29:38.868Z] Successfully created ./some-widget 
+[2021-09-08T17:29:38.870Z] Successfully created schema.json in ./some-widget/schema.json
+[2021-09-08T17:29:38.871Z] Successfully created config.json in ./some-widget/config.json
+[2021-09-08T17:29:38.871Z] Successfully created widget.html in ./some-widget/widget.html
+[2021-09-08T17:29:39.564Z] Starting widget-builder at http://localhost:8080!
+[2021-09-08T17:29:39.887Z] Socket connected.
+```
+
+Your default browser should open automatically and display a text widget on port 8080.
+
+## Continue development with `start`
+
+To spin up the development server for any existing widget, first ensure that it is located in a child directory of the root where you configured the corresponding environment.  Navigate to that environment root, then run the following command:
+
+```shell
+widget-builder start [[path-to/]widget-name]
+```
+**Sample output:**
+
+```shell
+[2021-09-08T17:29:39.564Z] Starting widget-builder at http://localhost:8080!
+[2021-09-08T17:29:39.887Z] Socket connected.
+```
+
+Your default browser should open automatically on port 8080.
+
+## Publish to store
+
+To publish a widget to your BigCommerce store, navigate to its environment root, then run the following command:
+
+```shell
+widget-builder publish [[path/to/]widget-name]
+```
+
+If you're interested in making your widgets available for download or sale on other stores, check out the [BigCommerce App Marketplace](https://www.bigcommerce.com/apps/toolswidgets/) and learn about [becoming a partner developer](https://developer.bigcommerce.com/api-docs/partner/becoming-a-partner).
```

Comment:
We don't have any provision for Widget in the current marketplace. We should check with the product on this one.

Reply:
Resolved offline.  TL;DR widgets are currently for sale in the marketplace.